src/controller/README.md

```diff
@@ -0,0 +1,68 @@
+# CHIP controller
+
+There are currently 4 implementations of the CHIP device controller with varying
+support for certain CHIP features.
+
+## Implementations
+
+### iOS
+
+The iOS chip-tool is located in [../darwin/CHIPTool](../darwin/CHIPTool).
+
+### Android
+
+The Android chip-tool is located in [../android/CHIPTool](../android/CHIPTool).
+
+### POSIX CLI
+
+The POSIX CLI chip-tool is located in
+[../../examples/chip-tool](../../examples/chip-tool).
+
+### Python
+
+The Python chip-device-ctrl is located in
+[../controller/python/](../controller/python).
+
+## Feature Overview
+
+| Onboarding | iOS | Android | CLI | Python |
+| ---------- | --- | ------- | --- | ------ |
+| Setup code | y?  | y?      | y   | y      |
+| QR code    | y   | y       | n   | n      |
+| NFC        | n   | y       | n   | n      |
+
+| Provisioning | iOS    | Android | CLI | Python |
+| ------------ | ------ | ------- | --- | ------ |
+| Soft-AP      | y      | y?      | y   | y      |
+| WiFi-BLE     | y?     | y?      | y   | y      |
```

Comment:
I haven't tested it, but it's implemented for Android for sure.

Reply:
That is correct, this is implemented in Android.
Thanks @Damian-Nordic

Reply:
SoftAP as well as WiFi-BLE or only the latter one?

Reply:
Only WiFi-BLE is implemented for Android. AFAIK, soft-AP plumbing isn't available in CHIP yet.